tests/transformations/loop_to_map_test.py

```diff
@@ -111,11 +119,6 @@ def apply_and_verify(sdfg, n):
 
     num_transformations = sdfg.apply_transformations(LoopToMap)
 
-    try:
-        os.remove("loop_to_map_test.txt")
```

Comment:
don't you want to still remove the temp file?

Reply:
This is not necessary anymore, as we are using a unique `tempfile.NamedTemporaryFile` every time the function is called, which is cleaned up after the test.